src/libraries/System.Numerics.Tensors/src/System.Numerics.Tensors.csproj

```diff
@@ -6,7 +6,6 @@
     <IsPackable>true</IsPackable>
     <PackageDescription>Provides support for operating over tensors.</PackageDescription>
     <GenAPIExcludeApiList>ReferenceAssemblyExclusions.txt</GenAPIExcludeApiList>
-    <ApiCompatValidateAssemblies>true</ApiCompatValidateAssemblies>
```

Comment:
@michaelgsharp unrelated change that I smuggled in. This property already defaults to true. No other library sets it explicitly.

Reply:
Yeah, I had it there so I could easily set it to false for offline work since we were adding so many new api's. I'll remove it.